Client/game_sa/CAnimManagerSA.h

```diff
@@ -175,6 +175,6 @@ class CAnimManagerSA : public CAnimManager
     // ped block, but in reality, it's playing our custom animation, and Of course, we can return run_wuzi
     // animation within the hook if we want to play it instead. Why run_wuzi? We can also use another animation,
     // but I've tested with this one mostly, so let's stick to this.
-    const char* m_kGateWayBlockName = "ped";
-    const char* m_kGateWayAnimationName = "run_wuzi";
+    static const char* const m_kGateWayBlockName;
+    static const char* const m_kGateWayAnimationName;
```

Comment:
These variables can be moved into cpp sources. There's no reason to keep them in a header file.